deps/internal/cmds/get/topology.go

```diff
@@ -164,6 +285,28 @@ func topologyCommand(
 				return err
 			}
 
+			if !(destinationModuleName == "") {
+				orgAndModule := parseName(req.GetLanguage(), destinationModuleName)
```

Comment:
Only after a rebase, I realized we don't parse this on CLI anymore. 

@mjpitz, Can you give me some more background on how this is changing with beta, so I could fix this?